e2e/__tests__/toThrowErrorMatchingInlineSnapshot.test.ts

```diff
@@ -74,14 +74,16 @@ test('cannot be used with .not', () => {
   const filename = 'cannot-be-used-with-not.test.js';
   const template = makeTemplate(`
     test('cannot be used with .not', () => {
-      expect('').not.toThrowErrorMatchingInlineSnapshot();
+      expect(() => { throw new Error('apple'); })
+        .not
+        .toThrowErrorMatchingInlineSnapshot();
     });
   `);
 
   {
     writeFiles(TESTS_DIR, {[filename]: template()});
     const {stderr, exitCode} = runJest(DIR, ['-w=1', '--ci=false', filename]);
-    expect(stderr).toMatch('.not cannot be used with snapshot matchers');
+    expect(stderr).toMatch('Snapshot assertion must not have not');
```

Comment:
Not entirely happy with this error message, what about
```suggestion
    expect(stderr).toMatch('Snapshot matchers cannot be used with .not');
```

Reply:
Yes, I will make the suggested change consistent change locally.